solr/core/src/java/org/apache/solr/handler/UpdateRequestHandler.java

```diff
@@ -146,6 +147,7 @@ protected Map<String, ContentStreamLoader> createDefaultLoaders(NamedList<?> arg
     registry.put("application/json", new JsonLoader().init(p));
     registry.put("application/csv", new CSVLoader().init(p));
     registry.put("application/javabin", new JavabinLoader(instance).init(p));
+    registry.put("application/cbor", CborStream.createLoader(p));
```

Comment:
Looking at nearby lines of code, this suggests that perhaps CborStream should be called "CborLoader"?